src/plugins/intel_cpu/src/transformations/cpu_opset/common/pass/convert_inverse_input_dtype.cpp

```diff
@@ -0,0 +1,53 @@
+// Copyright (C) 2018-2024 Intel Corporation
```

Comment:
This transformation is excessive. https://github.com/openvinotoolkit/openvino/pull/22834/files#diff-8180f616d8b55c262a606acbe6065b02b7e2802c5926ff5b44fc282cb36653cdR56 - this change is enough.

Reply:
Removed!